contracts/test/UnitTestVoteRecounter.sol

```diff
@@ -1,7 +1,8 @@
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.27;
 
-/// @dev FakeVoteRecounter is a VoteCounter contract for used unit tests
-contract FakeVoteRecounter {
+/// @dev UnitTestVoteRecounter.sol is a VoteCounter contract for used unit tests
```

Comment:
UnitTestVoteRecounter.sol -> UnitTestVoteRecounter
contract for used unit tests -> implementation for unit testing purposes